app/controllers/tag_controller.rb

```diff
@@ -568,4 +568,10 @@ def fetch_counts
   end
 
   def topic_tree; end
+
+  def subtopics
+nodes = Tag.find_by(name: "parent:#{params[:topic]}") # here, we look for this special tag marking a node as having a parent
+  .nodes
+  .collect(&:slug) # collect the "slugs" i.e. the unique part of the URL, which should correspond to a tagname
+    render "tags/subtopic", layout: false
```

Comment:
OK, so here, i think we also have to actually make the partial template, at: `/app/views/tags/_subtopics.html.erb`, and for now it can just print out the subtopics. If you'd like, you can look at the HTML you made in #11230 and use that?

Reply:
@jywarren : 
I created the file `_subtopics.html.erb` with the following code so as to only print the subtopics when pointed at a URL say `http://localhost:3000/tags/subtopic/one` or `http://localhost:3000/tags/subtopic/two`:

```ruby
<% @subtopics.each do |subtopic| %>
    <h1><%= subtopic %></h1>
<% end %>
```

However, it gives an error -

![image](https://user-images.githubusercontent.com/78212650/188291739-bf4052ae-0030-4b76-94e7-536952eabd96.png)

Is it because the tags "one" or "two" do not have any specified subtopics as such? Let me know if I am wrong. 

Thanks!